src/main/scala/io/shiftleft/js2cpg/core/Js2Cpg.scala

```diff
@@ -197,9 +197,7 @@ class Js2Cpg {
     }
 
     if (config.includeConfigs) {
-      val freshJsons  = FreshJsonParser.findImportMapPaths(config, includeDenoConfig = true).map(_.normalize)
-      val configFiles = this.configFiles(config, CONFIG_FILES).filter(p => freshJsons.contains(p._1.normalize))
```

Comment:
This was wrong anyway. Without any fresh json we would never pick up config files.

Reply:
Yes, I think it was supposed to include the deno/fresh config files even without flag. But that's not what it did. And that the default has changed, this complication isn't needed anymore.